transcripts/496-scaf-deploys.txt

```diff
@@ -44,7 +44,7 @@
 
 00:02:14 This episode has something unique and special.
 
-00:02:17 I thought it'd be fun if I could come up with a song that was about deploying code, about Kubernetes, about the cloud, all the different things that Calvin's gonna come on and tell us all about.
+00:02:17 I thought it'd be fun if I could come up with a song that was about deploying code, about Kubectl, about the cloud, all the different things that Calvin's gonna come on and tell us all about.
```

Comment:
Hi @manikanta-hitunik-com this still has problems, on line 5 and 47, it changed Kubernetes to Kubectl, but it is Kubernetes in the audio. But it does fix it and go back to kubectl on line 1609. It's not just a find-replace of Kubernetes -> Kubectl. Some were good and some were bad. That's what. I meant about needing to listen to the audio to see which is which.